### PR TITLE
Add SQLite backend is_in_memory_db() functions

### LINKED ISSUE #930: django.db.backends.sqlite3.base.DatabaseWrapper.is_in_memory_db is missing
Please add [`django.db.backends.sqlite3.base.DatabaseWrapper.is_in_memory_db`](https://github.com/django/django/blob/6f311c7e35090957dd93783b71c5d3af7d5329d4/django/db/backends/sqlite3/base.py#L339)`() -> `[`bool`](https://github.com/django/django/blob/6f311c7e35090957dd93783b71c5d3af7d5329d4/django/db/backends/sqlite3/creation.py#L13-L17).

### PR DESCRIPTION
**Pull request by adamchainz**
# I have made things!

Sources:

* `DatabaseWrapper`: https://github.com/django/django/blob/c01e76c95caa1bd2995f1b6c186cbe8726c92ef9/django/db/backends/sqlite3/base.py#L351
* `DatabaseCreation`: https://github.com/django/django/blob/c01e76c95caa1bd2995f1b6c186cbe8726c92ef9/django/db/backends/sqlite3/creation.py#L13-L17

## Related issues

Fixes #930. @wkschwartz would you like to review?

### CODE REVIEW

File: django-stubs/db/backends/sqlite3/creation.pyi [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
+from typing import Union
+
 from django.db.backends.base.creation import BaseDatabaseCreation
 from django.db.backends.sqlite3.base import DatabaseWrapper
 
 class DatabaseCreation(BaseDatabaseCreation):
     connection: DatabaseWrapper
+
+    @staticmethod
+    def is_in_memory_db(database_name: Union[str, PathLike]) -> bool: ...
```
Comment (sobolevn) [inline django-stubs/db/backends/sqlite3/creation.pyi]:
```suggestion
    def is_in_memory_db(database_name: Union[str, PathLike[str]]) -> bool: ...
```

**Review by sobolevn — APPROVED** [review]
Thanks!

**Review by wkschwartz — APPROVED** [review]
Thanks!

File: django-stubs/db/backends/sqlite3/creation.pyi (hunk 1 above)
Comment (adamchainz) [reply django-stubs/db/backends/sqlite3/creation.pyi]: Ah yes, thanks!

**Pull request merged**